### ISSUE TITLE
Typescript Definition for KeyboardAwareScrollView seems to be missing in v5.6.0

### ISSUE THREAD

**Issue by girishjjain** [wix/react-native-ui-lib#746]
I upgraded to latest version 5.6.0 and got Typescript error for KeyboardAwareScrollView import statement `import { KeyboardAwareScrollView } from 'react-native-ui-lib';`. 

Error:
`Module '"../node_modules/react-native-ui-lib"' has no exported member 'KeyboardAwareScrollView'.ts(2305)`

**Comment by girishjjain** [comment]
I also see errors for almost every component, for example, using `<View paddingH-15 paddingV-5>` gives following error:

`Type '{ children: Element[]; "paddingH-15": true; "paddingV-5": true; }' is missing the following properties from type 'Pick<Pick<PropsTypes, "center" | "style" | "row" | "spread" | "centerH" | "centerV" | "left" | "right" | "top" | "bottom" | "padding" | "paddingL" | "paddingT" | "paddingR" | ... 77 more ... | "onTouchEndCapture">, "center" | ... 89 more ... | "onTouchEndCapture">': accessibilityValue, onAccessibilityEscapets(2739)`

**Comment by ethanshar** [comment]
@girishjjain 
I've release a HF (v5.6.1) for the first issue with the missing component. 

Regarding the second issue, unfortunately I couldn't reproduce. 
I added in my code the following `<View paddingH-15 paddingV-5>` and didn't get any similar error. 
You said you're getting this a lot, can you paste other snippets so we can investigate. 

Thanks.

**Comment by girishjjain** [comment]
Thanks @ethanshar for HF, the error with `KeyboardAwareScrollView` import is fixed. For `<View>`, I tried below:

```
import React from 'react';
import { View } from 'react-native-ui-lib';

export default class TestView extends React.Component {

  public render() {
    return(
      <View>       
      </View>
    );
  }
}
```

and got following error:
```
src/ui/TestView.tsx:8:8 - error TS2739: Type '{ children: never[]; }' is missing the following properties from type 'Pick<Pick<PropsTypes, "row" | "left" | "right" | "center" | "top" | "bottom" | "style" | "hitSlop" | "onLayout" | "pointerEvents" | "removeClippedSubviews" | "testID" | ... 79 more ... | "inaccessible">, "row" | ... 89 more ... | "inaccessible">': accessibilityValue, onAccessibilityEscape

8       <View>
```

I am using typescript version:
```
"typescript": "^3.8.3"
```

**Comment by Macrow** [comment]
I have the same error. 
:(

**Comment by ethanshar** [comment]
Hey,
Issues mentioned in this issue should be resolved in our latest release `v5.7.0`
for other typescript issues, please open a new issue. 
Thanks 🙏
